pkg/loki/loki.go

```diff
@@ -634,6 +629,9 @@ func (t *Loki) setupModuleManager() error {
 	mm.RegisterModule(QueryFrontendTripperware, t.initQueryFrontendTripperware, modules.UserInvisibleModule)
 	mm.RegisterModule(QueryFrontend, t.initQueryFrontend)
 	mm.RegisterModule(RulerStorage, t.initRulerStorage, modules.UserInvisibleModule)
+	mm.RegisterModule(QueryLimiter, t.initQueryLimiter, modules.UserInvisibleModule)
```

Comment:
I think we should make the limiter configurable. What do you think?

Reply:
I'm not sure what configurable means in this context.